nntrainer/models/neuralnet.cpp

```diff
@@ -180,24 +173,22 @@ int NeuralNetwork::initialize() {
   model_graph.setBatchSize(
     std::get<props::TrainingBatchSize>(model_flex_props));
 
-  auto &input_layer_prop =
+  auto &input_conns =
     std::get<std::vector<props::InputConnection>>(model_props);
   auto &label_layer_prop =
     std::get<std::vector<props::LabelLayer>>(model_props);
```

Comment:
LabelLayer to be LabelConnection?

Reply:
This gotta be, but label is not connection so it will stay same until https://github.com/nnstreamer/nntrainer/pull/1738#discussion_r761645143 is being implemented